modules/policy-incident.adoc

```diff
@@ -5,8 +5,9 @@
 [id="policy-incident_{context}"]
 = Incident and operations management
 
-
-This documentation details the Red Hat responsibilities for the {product-title} managed service.
+This documentation details the Red Hat responsibilities for the {product-title} managed service. 
+The cloud provider is responsible for protecting the hardware infrastructure that runs the services offered by the cloud provider.
+The customer is responsible for incident and operations management of customer application data and any custom networking the customer may have configured for the cluster network or virtual network.
```

Comment:
```suggestion
The customer is responsible for incident and operations management of customer application data and any custom networking the customer might have configured for the cluster network or virtual network.
```

Reply:
Just changed this to "the customer has configured"